games/hedgewars/Portfile

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.0
+
+name                    hedgewars
+version                 0.9.22
+categories              games
+platforms               darwin
+license                 GPL-2
+maintainers             {gmail:ken.cunningham.webuse @kencu} openmaintainer
+
+description             Funny turn-based artillery game, featuring fighting Hedgehogs!
+long_description        ${description}
+homepage                https://www.hedgewars.org/
+
+master_sites            https://www.hedgewars.org/download/releases/
+use_bzip2               yes
+distname                ${name}-src-${version}
+
+checksums               rmd160  359fd026643e69446121b94ba301646812353c19 \
+                        sha256  b699c8971ff420c3edd6533527ae2e99040f1e79207c9140826945bcf0e62192
+
+cmake.out_of_source     yes
+
+depends_build-append    port:pkgconfig \
+                        port:ghc \
+                        port:hs-vector \
+                        port:hs-bytestring-show \
+                        port:hs-network \
+                        port:hs-dataenc \
+                        port:hs-hslogger \
+                        port:hs-utf8-string \
+                        port:hs-sha \
+                        port:hs-entropy \
+                        port:hs-zlib \
+                        port:hs-random
+                    
+depends_lib-append      path:bin/ffmpeg:ffmpeg \
+                        port:libogg \
+                        port:libsdl \
+                        port:libsdl_image \
+                        port:libsdl_ttf \
+                        port:libsdl_mixer \
+                        port:libsdl_net \
+                        port:zlib \
+                        port:libpng \
+                        port:glew \
+                        port:qt4-mac
+
+# don't use the sparkle selfupdate mechanism; use macports update process instead
+configure.args-append   -DNOAUTOUPDATE=1
+
+# don't add all of the macports support libraries into the app bundle
+configure.args-append   -DSKIPBUNDLE:BOOL=ON
+
+configure.args-append   -DQT_QMAKE_EXECUTABLE:FILEPATH=${prefix}/libexec/qt4/bin/qmake
```

Comment:
It didn't seem to like having the qmake Portgroup included -- caused errors during configuration.
```
Command failed:  cd "/opt/local/var/macports/build/_opt_macports-ports_games_hedgewars/hedgewars/work/build" && /opt/local/libexec/qt4/bin/qmake -DCMAKE_INSTALL_PREFIX='/opt/local' PREFIX=/opt/local QMAKE_CC=/opt/local/bin/clang-mp-3.9 QMAKE_CXX=/opt/local/bin/clang++-mp-3.9 QMAKE_OBJC=/opt/local/bin/clang-mp-3.9 QMAKE_CFLAGS="-Os " QMAKE_CXXFLAGS="-Os -stdlib=libc++ " QMAKE_LFLAGS="-L/opt/local/lib -Wl,-headerpad_max_install_names" QMAKE_LINK_C=/opt/local/bin/clang-mp-3.9 QMAKE_LINK_C_SHLIB=/opt/local/bin/clang-mp-3.9 QMAKE_LINK=/opt/local/bin/clang++-mp-3.9 QMAKE_LINK_SHLIB=/opt/local/bin/clang++-mp-3.9 -DCMAKE_BUILD_TYPE=Release -DCMAKE_BUILD_WITH_INSTALL_RPATH=ON -DCMAKE_C_COMPILER="$CC" -DCMAKE_COLOR_MAKEFILE=ON -DCMAKE_CXX_COMPILER="$CXX" -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_INSTALL_NAME_DIR=/opt/local/lib -DCMAKE_INSTALL_RPATH=/opt/local/lib -DCMAKE_MODULE_PATH=/opt/local/share/cmake/Modules -DCMAKE_SYSTEM_PREFIX_PATH="/opt/local;/usr" -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev -DNOAUTOUPDATE=1 -DSKIPBUNDLE:BOOL=ON -DBUILD_ENGINE_C:BOOL=ON -DCMAKE_C_FLAGS_RELEASE="-DNDEBUG" -DCMAKE_CXX_FLAGS_RELEASE="-DNDEBUG" -DCMAKE_OSX_ARCHITECTURES="x86_64" -DCMAKE_OSX_DEPLOYMENT_TARGET="10.6" -DCMAKE_OSX_SYSROOT="/" /opt/local/var/macports/build/_opt_macports-ports_games_hedgewars/hedgewars/work/hedgewars-src-0.9.22 
Exit code: 1
Error: Failed to configure hedgewars: configure failure: command execution failed
Error: See /opt/local/var/macports/logs/_opt_macports-ports_games_hedgewars/hedgewars/main.log for details.
Error: Follow https://guide.macports.org/#project.tickets to report a bug.
Error: Processing of port hedgewars failed
```